setup.cfg

```diff
@@ -1,9 +1,2 @@
 [aliases]
```

Comment:
Why is the purpose of the setup.cfg here if you already have `tox.ini` giving instructions on the test

Reply:
This is settings for setuptools.

Reply:
FYI, including this setting and `setup.py test` is [deprecated and will trigger a warning](https://github.com/pypa/setuptools/issues/1684) in new versions of setuptools, and will eventually be removed.